.github/workflows/build-push.yml

```diff
@@ -15,9 +15,36 @@ concurrency:
   cancel-in-progress: ${{ github.event_name == 'pull_request' }}
 
 jobs:
+  test-lint:
+    name: Test with Python ${{ matrix.python-version }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        python-version:
+          - 3.9
+          - 3.10
+
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install dependencies
+      run: pip install -r requirements-dev.txt
+
+    - name: Lint with flake8
```

Comment:
Have you considered using [ruff](https://github.com/charliermarsh/ruff) instead of flake8?

Reply:
Nooice, done, thanks!